docs/configuration.md

```diff
@@ -19,7 +19,12 @@
 | MetricScrapingPeriodSeconds  | int       | > 0                                                                 | 15                  | Period of Prometheus metrics scrapping                                               |
 | GRPCConnectionTimeoutSeconds | int       | > 0                                                                 | 60                  | Timeout for establishing a gRPC connection                                           |
 | GRPCFunctionTimeoutSeconds   | int       | > 0                                                                 | 90                  | Maximum time given to function to execute[^4]                                        |
-| DAGMode             | bool      | true/false                                                          | false               | Sequential invocation of all functions one after another                                                    |
+| DAGMode                      | bool      | true/false                                                          | false               | Parallel invocation of DAG with each function acting as entry function                                                                                                   |
```

Comment:
This is okay for the description of the config option, but we need proper documentation of this feature in the docs. Please consider writing down the generation and invocation processes of the DAG so the user can understand the output results without looking at the code.

Reply:
Have added some simple descriptions and explanation to run in section ```Workflow Invocation``` in  ```docs/loader.md```. Is it sufficient? Or is a more comprehensive one required? (Explaining logic behind workflow generation, going through the invocations step by step etc) Thanks!

Reply:
Please add an explanation of the whole process for DAG parameter generation (width, depth, invocation duration, count, IAT)

Reply:
Can I check if the explanation is sufficient? Have mentioned about certain parameters (frequency, IAT) briefly as they are not newly generated and unchanged from the original implementation.